### PR TITLE
Get the barest bones of webdriver integration ready for keeners.

### PR DESCRIPTION
**Pull request by jdm**
Requires https://github.com/jgraham/webdriver-rust/pull/6.

### CODE REVIEW

**Comment by jdm** [comment]
This will allow me to file issues for people to work on integration in pieces, and will force us to keep webdriver-rust up to date with rustc upgrades.

**Comment by metajack** [comment]
Assigned @pcwalton.

File: components/util/opts.rs [hunk 1]
```diff
@@ -226,6 +231,7 @@ pub fn from_cmdline_args(args: &[String]) -> bool {
         getopts::optflag("z", "headless", "Headless mode"),
         getopts::optflag("f", "hard-fail", "Exit on task failure instead of displaying about:failure"),
         getopts::optflagopt("", "devtools", "Start remote devtools server on port", "6000"),
+        getopts::optflagopt("", "webdriver", "Start remote devtools server on port", "7000"),
```
Comment (pcwalton) [inline components/util/opts.rs]: `s/devtools/webdriver/`?

**Comment by pcwalton** [comment]
r+

**Pull request merged**